Makefile

```diff
@@ -208,3 +208,10 @@ gitserver:
 		--platform $(PLATFORMS) \
 		-t $(GITSERVER_IMAGE) \
 		--push .
+
+test-e2e:
+test-%:
+	kubectl -n numaplane-system delete pod localgitserver-0 --ignore-not-found=true
```

Comment:
I believe it's an entire StatefulSet that needs to be deleted, right? not just the Pod?

Reply:
and the `Service` as well.
Perhaps, since below you run `kubectl apply -n numaplane-system -k ./tests/e2e-gitserver`, does it work to first do `kubectl delete -n numaplane-system -k ./tests/e2e-gitserver`? That might be nice and clean.

Reply:
Just tried this, works just fine. Will make change.